README.md

```diff
@@ -110,13 +114,13 @@ You should be able to do the same in Mac or Windows but note that setting up a H
     Retype new password:
     passwd: all authentication tokens updated successfully.
     ```
-* As root edit /etc/sudoers and allow openbdre to perform `sudo`. Below will do it
+* As root edit `/etc/sudoers` and allow openbdre to perform `sudo`. Below will do it
 
     ```shell
     echo "openbdre ALL=(ALL) NOPASSWD:ALL" >> /etc/sudoers
     ```
 
-* Login to the HDP Sandbox with the newly created openbdre user. You can perform a **su openbdre** to switch to this account. Please make sure you are not root user beyond this point.
+* Login to the HDP Sandbox with the newly created openbdre user. You can perform a `su openbdre` to switch to this account. Please make sure you are not root user beyond this point.
```

Comment:
Probably don't need a code style highlight here as the code highlight is done immediately below